_publications/2020_vmcai_gbhr.md

```diff
@@ -30,17 +30,17 @@ year: 2020
 bibentry: inproceedings
 bib:
   booktitle: Proceedings of the 21st International Conference on Verification, Model Checking, and Abstract Interpretation (VMCAI)
-  series:
-  volume:
-  publisher:
-  editor:
-  pages:
-  doi:
+  series: Lecture Notes in Computer Science
+  volume: 11990
+  publisher: Springer
+  editor: Beyer, Dirk and Zufferey, Damien
```

Comment:
To make things consistent, this should be `Dirk Beyer and Damien Zufferey`. Otherwise, it looks good.